imports/plugins/core/orders/server/no-meteor/resolvers/Refund/index.js

```diff
@@ -0,0 +1,10 @@
+import amount from "./amount";
+import createdAt from "./createdAt";
+
+export default {
+  _id: (node) => node._id,
```

Comment:
You need to encode this ID here. May need a new transform for "reaction/refund" namespace.

You should also add `paymentId` here and encode that (and add it to GraphQL schema).